modules/standard/Random.chpl

```diff
@@ -1134,17 +1155,17 @@ module Random {
       }
 
       /*
-     Returns a random sample from a given 1-D domain, ``x``.
+     Returns a random sample from a given N-D domain, ``x``.
 
-     :arg x: a 1-D dom with values that will be sampled from.
+     :arg x: a N-D dom with values that will be sampled from.
      :arg size: An optional integral value specifying the number of elements to
                 choose, or a domain specifying the dimensions of the
                 sampled array to be filled, otherwise a single element will be
                 chosen.
      :arg replace: an optional ``bool`` specifying whether or not to sample with
                    replacement, i.e. elements will only be chosen up to one
                    time when ``replace=false``.
-     :arg prob: an optional 1-D array that contains probabilities of choosing
+     :arg prob: an optional N-D array that contains probabilities of choosing
```

Comment:
I think we can remove dimensionality from all these statements. `an optional array` is sufficient, to me.